t/app/controller/auth_social.t

```diff
@@ -45,6 +46,8 @@ foreach ($body->id, $body2->id, $body3->id) {
 my $resolver = Test::MockModule->new('Email::Valid');
 my $social = Test::MockModule->new('FixMyStreet::App::Controller::Auth::Social');
 $social->mock('generate_nonce', sub { 'MyAwesomeRandomValue' });
+my $tilma = t::Mock::Tilma->new;
+LWP::Protocol::PSGI->register($tilma->to_psgi_app, host => 'tilma.mysociety.org');
```

Comment:
As discussed in slack, "with two lines, for live and staging" - the staging line is missing here, but it is being called by Brent.pm because STAGING_SITE is true.

Reply:
So can I just add another host? By repeating the line but with staging.tilma...?

Reply:
Yes